modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/gradle.properties

```diff
@@ -2,6 +2,8 @@
 com.liferay.portal.tools.rest.builder.version=1.0.93
 #elseif (${liferayVersion.startsWith("7.3")})
 com.liferay.portal.tools.rest.builder.version=1.0.212
+#elseif (${liferayVersion.startsWith("7.4")})
+com.liferay.portal.tools.rest.builder.version=1.0.407
 #else
 com.liferay.portal.tools.rest.builder.version=@com.liferay.portal.tools.rest.builder.version@
```

Comment:
@kyle-miho for this one, we actually just want to let Workspace set the version when using newer versions, which means this property is not needed. 

What we do want here is a commented out line demonstrating that they _could_ set the version explicitly if they wanted to. 

So keep the older branches the same, but have the newer branches just use a commented-out line.

Reply:
I added an extra pound sign because of the pound sign already existing for the elseif statements. Im not sure if thats the best way to do it? It still works locally for me